modules/ROOT/pages/configure-client-management-azure.adoc

```diff
@@ -47,7 +47,9 @@ Use the following properties when you are using the Authorization Code grant typ
 
 Anypoint Platform does not support custom scopes. Use the default scope for Microsoft Graph. 
 
+=== Grant Types === 
 
+Microsoft doesn't support specifying grant types when https://docs.microsoft.com/en-us/graph/api/application-post-applications?view=graph-rest-1.0&tabs=http[creating a new application] and it supports "authorization_code" and "client_credentials" by default. 
```

Comment:
```suggestion
When https://docs.microsoft.com/en-us/graph/api/application-post-applications?view=graph-rest-1.0&tabs=http[creating a new application], Microsoft supports `authorization_code` and `client_credentials` grant types by default. Microsoft does not support specifying other grant types.
```

Reply:
Thanks for the suggestion. So the confusion customer had here is that they may want to restrict to "authorization_code" only but in fact we don't have control of what's supported at Microsoft Azure. 

Mentioning these two is because we know for sure these 2 are supported by default. There may be more that's supported like "Implicit" grant type but we have not tested it yet. 

Hopefully I'm making sense.. Let's maybe grab 10min if needed

Reply:
```suggestion
For a list of supported grant types, see the https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow[Microsoft identity platform documentation].
```